uber/menu.py

```diff
@@ -51,27 +51,26 @@ def render_items_filtered_by_current_access(self):
         """
         out = {}
 
-        allowed = True
-        if self.access:
-            allowed = False
-            for access_level in listify(self.access):
-                if access_level in sa.AdminAccount.access_set():
-                    allowed = True
-                    break
-
-        if not allowed:
+        if self.access and set(listify(self.access)).isdisjoint(sa.AdminAccount.access_set()):
```

Comment:
@kitsuta with the excellent one-liner.  when writing that other code I was thinking 'I bet there's some python trickery to do this in one line'